src/runtime/manifest.ts

```diff
@@ -529,8 +529,11 @@ ${e.message}
       throw processError(e, false);
     }
     dumpErrors(manifest);
-    if (options.throwImportErrors && manifest.errors.length > 0) {
-      throw manifest.errors[0];
+    if (options.throwImportErrors) {
```

Comment:
Should this be in options? Might be better to have it as a global flag (in src/runtime/flags.ts) so that we can turn it on and off globally.